FluentUI/src/main/java/com/microsoft/fluentui/util/DuoSupportUtils.kt

```diff
@@ -12,38 +13,38 @@ import java.lang.Exception
  */
 object DuoSupportUtils {
     const val DUO_HINGE_WIDTH = 84
+    const val COLUMNS_IN_START_DUO_MODE = 3
+    const val COLUMNS_IN_END_DUO_MODE = 4
 
     @JvmStatic
-    fun isDeviceSurfaceDuo(activity: Activity) = ScreenHelper.isDeviceSurfaceDuo(activity)
+    fun isDeviceSurfaceDuo(activity: AppCompatActivity) = ScreenHelper.isDeviceSurfaceDuo(activity)
```

Comment:
is AppCompatActivity required or Activity is enough as some old base apps  might not be using Appcompat yet.
I know office has just shifted

Reply:
Using activity now